src/main/java/com/epam/reportportal/junit5/ReportPortalExtension.java

```diff
@@ -786,7 +788,25 @@ protected FinishTestItemRQ buildFinishTestItemRq(@Nonnull ExtensionContext conte
 	 */
 	protected String createStepName(ExtensionContext context) {
 		String name = context.getDisplayName();
-		return name.length() > 1024 ? name.substring(0, 1021) + "..." : name;
+		String defaultValue = name.length() > 1024 ? name.substring(0, 1021) + "..." : name;
```

Comment:
The same logic is repeated 3 time in one method, please consider refactoring or moving it into separate method.

Reply:
Extract to private method this logic